pkg/util/iptables/iptables.go

```diff
@@ -332,7 +333,7 @@ func (runner *runner) SaveInto(table Table, buffer *bytes.Buffer) error {
 	// creates a new buffer, redirects stdout and stderr to it and also
 	// calls Run()].
 	cmd.SetStdout(buffer)
-	cmd.SetStderr(buffer)
+	cmd.SetStderr(ioutil.Discard)
```

Comment:
@luksa it looks like this will just drop stderr on the floor even if the command resulted in an error. Instead, could we put stderr into another buffer, and return *that* buffer if cmd.Run() returns an error?

Reply:
@dcbw Done. PTAL